docs/getting-started/getting_started_14.md

```diff
@@ -0,0 +1,110 @@
+In order to use icons with GTK, we have two options : 
+
+- Use icons bundled with GTK.
+- Bundle SVG icons within our app resources.
+
+### GTK bundled icons
+
+To use GTK bundle icons, we just need to know the name of the icon we want to use. We can use the "GTK Icon Browser" app to see the icons installed on our system.
+
+To install it, run :
+
+=== "Ubuntu/Debian"
+
+    ```bash
+    apt-get install gtk-3-examples
+    ```
+
+=== "Fedora/RHEL"
+
+    ```bash
+    dnf install gtk3-devel
```

Comment:
There's a gtk4-icon-browser too. In Ubuntu/Debian, it is in the package `gtk-4-examples`, in Fedora it is in `gtk4-devel-tools`.